constellations/gemini-user/src/account/account.sass

```diff
@@ -0,0 +1,302 @@
+
+        .tab-content
+            ul
+                display: flex
+                flex-direction: row
+                flex-wrap: wrap
+                border-bottom: 1px solid $colorWhiteLight
+                text-align: left
+
+                li
+                    padding: $spacing1x 0
+
+        .tab-content#payment-tab
+            ul
+                li:first-child
+                    width: 100%
+                    padding-bottom: 0
+                li
+                    width: calc(100% / 3)
+
+        .tab-content#reg-tab-main
+            ul
+                li
+                    width: 50%
+
+                    .classList-item
+                        display: flex
+                        flex-direction: column
+                        align-items: baseline
+                        margin: $spacing2x 0
+
+                        span:last-child
+                            color: #888888
+
+                    .classList-item:first-child
+                        margin-top: 0
+                    .classList-item:last-child
+                        margin-bottom: 0
+
+            ul.afh-table
+                .li-large
+                    flex: 1
+                    padding-left: 50%
+
+        .tab-content#reg-tab-all
+            div.space-between
+                align-items: center
+
+                h2
+                    margin: 0
+
+            ul
+                padding: $spacing1x 0
+
+                li:first-child
+                    width: 100%
+
+                .li-large
+                    flex: 1
+
+                li:last-child
+                    max-width: 40%
+                    text-align: right
+
+
+        .tab-content#settings-tab
+            ul
+                padding: $spacing2x 0
+                .li-med
+                    width: 50%
+                .li-large
+                    flex: 1
+
+            ul.users-table
+                li:last-child
+                    padding-left: 50%
+
+            input
+                width: calc(100% - 16px)
+                /* not a great solution for overflowing padding, but it works */
+
+            .password-buttons
+                justify-content: space-between
+
+        ul.hide-mobile
+            display: none
+
+        .vertical-mobile
+            display: flex
+            flex-direction: column
+
+
+    .space-between
+        display: flex
+        justify-content: space-between
+        align-items: center
+
+    ul.header
+        border-bottom: 1px solid $colorGrayLight
+
+    ul
+        line-height: $lineHeight1
+
+    ul.no-border
+        border: none
+
+    .password-buttons
+        margin-top: $spacing2x
+
+    button
+        border-radius: $spacing1x
+
+    .btn-cancel
+        background-color: #787878
+
+    input
+        padding: $spacing1x
+        border: 1px solid $colorGrayLight
+        border-radius: $spacing1x
+
+    a:hover
+        cursor: pointer
+
+    .red
+        color: #d91211
```

Comment:
can you use the color defined in `_constants.sass` (or define it yourself there? like `colorRed`)